### PR TITLE
Release 1.4.19

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR
MAT-7945 add improvement notation description to group

### IF THIS IS A RELASE, PR MUST BE NAMED `Release x.y.z`, where x.y.z is the new version

Jira Ticket: [MAT-7945](https://jira.cms.gov/browse/MAT-7945)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by adongare — APPROVED** [review]

File: src/Measure.ts [hunk 1]
```diff
@@ -91,6 +91,7 @@ export interface Group {
   groupDescription?: string;
   rateAggregation?: string;
   improvementNotation?: string;
+  improvementNotationDescription?: string;
```
Comment (adongare) [inline src/Measure.ts:94]: because this is a release, package json needs to be updated as well for 1.4.19

Comment (adongare) [reply src/Measure.ts:94]: nvm, you did. did not refresh the page.

**Review by adongare — APPROVED** [review]
